libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Using DocFormatProvider
+page_title: Using DocFormatProvider
+description: Import from Word 97 - Word 2003 Binary file format Using DocFormatProvider
+slug: radwordsprocessing-formats-and-conversion-doc-docformatprovider
+tags: using,docformatprovider
+published: True
+position: 2
+---
+
+# Using DocFormatProvider
+
+**DocFormatProvider** makes it easy to import a **RadFlowDocument** from DOC format, preserving the entire document structure and formatting.
```

Comment:
**DocFormatProvider** makes it easy to import a DOC file into a **RadFlowDocument**, preserving the entire document structure and formatting.